pkg/framework/webhooks.go

```diff
@@ -81,7 +81,7 @@ func DeleteMutatingWebhookConfiguration(c client.Client, webhookConfiguraiton *a
 
 // UpdateMutatingWebhookConfiguration updates the specified mutating webhook configuration
 func UpdateMutatingWebhookConfiguration(c client.Client, updated *admissionregistrationv1.MutatingWebhookConfiguration) error {
-	return wait.PollImmediate(RetryShort, WaitShort, func() (bool, error) {
+	return wait.PollImmediate(RetryShort, WaitMedium, func() (bool, error) {
```

Comment:
can you elaborate what's motivating this change? can you share CI failures links that support that motivation and that this change would mitigate?

Reply:
https://github.com/openshift/cluster-api-actuator-pkg/pull/183#discussion_r465690265